CommonTools/PileupAlgos/plugins/PuppiProducer.h

```diff
@@ -51,6 +51,7 @@ class PuppiProducer : public edm::stream::EDProducer<> {
   bool fUseDZ;
   float fDZCut;
   double fPtMaxCharged;
+  double fEtaMaxCharged;
   double fPtMaxPhotons;
   double fEtaMaxPhotons;
   double fPtMaxNeutrals;
```

Comment:
```fPtMaxNeutrals``` and ```fPtMaxNeutralsStartSlope``` are not used here, only in PuppiContainer: you can remove from PuppiProducer then

Reply:
done